src/Examples/Flowsheets/HDA_flowsheet_with_costing.ipynb

```diff
@@ -0,0 +1,1598 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# HDA Flowsheet Costing\n",
+    "\n",
+    "\n",
+    "## Note\n",
+    "\n",
+    "This tutorial will demonstrate adding rigorous process costing to the two HDA examples, the basic [HDA with Flash](../../Tutorials/Basics/HDA_flowsheet_solution.ipynb) and a comparison with the [HDA with Distillation](HDA_flowsheet_with_distillation_solution.ipynb).\n",
```

Comment:
I am not sure we want to use the term "rigorous" here - I would say something like:

" demonstrate adding capital and operating cost to the two HDA examples"